src/api/app/controllers/webui/packages/build_reason_controller.rb

```diff
@@ -18,6 +17,8 @@ def index
       private
 
       def set_package
```

Comment:
Hm, this is more or less the same what require_package does in the webui controller. We should get rid of this one soon (not in this PR though).